system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
+Documentation=man:netdata
+Documentation=file:///usr/share/doc/netdata/html/index.html
+Documentation=https://github.com/firehol/netdata
 After=network.target httpd.service squid.service nfs-server.service mysqld.service named.service postfix.service
+Wants=network-online.target
+ConditionPathExists=/etc/netdata/netdata.conf
 
 [Service]
 Type=simple
+Environment="netdata_LOG_LOCATION=/var/log/netdata/log"
+ExecStart=@sbindir_POST@/netdata -D
+ExecReload=@sbindir_POST@/netdata reload
+TimeoutStopSec=60
+KillMode=mixed
+KillSignal=SIGTERM
+OOMScoreAdjust=-900
```

Comment:
We had a bug bug report about the OOM being too eager to kill Netdata:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=848642

> We don't want to have our monitoring running but the application servers crashed

Well... I've seen monitoring systems being configured to survive applications when they are used to investigate incidents and perform post-mortem.
Maybe limiting the maximum amount of memory used by Netdata would be a good safety measure?